src/engraving/dom/note.cpp

```diff
@@ -3816,4 +3820,9 @@ bool Note::negativeFretUsed() const
 {
     return configuration()->negativeFretsAllowed() && m_fret < 0;
 }
+
+int Note::stringOrLine() const
+{
+    return staff()->staffType(tick())->isTabStaff() ? string() * 2 : line();
```

Comment:
Wait, why `string() * 2`?

Reply:
The 6 strings are numbered 1 to 6 whereas the 5 stave lines are numbered 1 to 10 including spaces.  This method is used in layout methods which are built around lines and assume even numbers are spaces and odd are lines.  
I could make this more clear - rename it `staveLine()` or just leave a comment?

Reply:
Aha! Good point. No I think the name is fine, but definitely worth a comment :)